projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -451,10 +488,10 @@ private function build_checksum_query( $range_from = null, $range_to = null, $fi
 		$join_statement = '';
 		if ( $this->parent_table ) {
 			$parent_table_obj    = new Table_Checksum( $this->parent_table );
-			$parent_filter_query = $parent_table_obj->build_filter_statement( $range_from, $range_to, null, 'parent_table' );
+			$parent_filter_query = $parent_table_obj->build_filter_statement( $range_from, $range_to, null, 'parent_table', true );
```

Comment:
revert the addition of $exclude_ranges.

Instead simply pass null for $range_from and $range_to for the parent query..